api/src/db/migrations/v145_remove_nonprofit_onboarding.ts

```diff
@@ -0,0 +1,805 @@
+import { v144Business, v144UserData } from "@db/migrations/v144_add_raffle_bingo_games";
+import { randomInt } from "@shared/intHelpers";
+
+export const migrate_v144_to_v145 = (v144Data: v144UserData): v145UserData => {
+  return {
+    ...v144Data,
+    businesses: Object.fromEntries(
+      Object.values(v144Data.businesses)
+        .map((business: v144Business) => migrate_v144Business_to_v145Business(business))
+        .map((currBusiness: v145Business) => [currBusiness.id, currBusiness])
```

Comment:
you will have to update it to v147 now.. mine just went in with 145 from a previous commit and mine is 146